Makefile

```diff
@@ -51,19 +53,22 @@ restart-acp:
 .PHONY: lint
 lint: start-runner
 	docker exec quickstart-runner sh -c "golangci-lint run --fix --deadline=5m ./..."
+	make stop-runner
 
 .PHONY: clean
 clean: 
-	docker-compose -f docker-compose.acp.local.yaml -f docker-compose.build.yaml down --remove-orphans
+	docker-compose -f docker-compose.build.yaml down --remove-orphans
```

Comment:
is this also terminating ACP?

Reply:
This is a good question.
Locally `--remove-orphans` flag will do it, but on Jenkins I'm not sure - will check it.
E.g.: 
for  `quickstart-runner` container is not working remotely (locally is ok)

Reply:
It is 
`Removing orphan container "quickstart-acp"` 
![image](https://user-images.githubusercontent.com/72209717/161963915-c309506f-86f1-4f2d-9008-1945886ae425.png)